example/example_util.h

```diff
@@ -17,7 +17,28 @@
 
 #include <string>
 #include <parquet/parquet.h>
+#include <stdio.h>
 
 bool GetFileMetadata(const std::string& path, parquet::FileMetaData* metadata);
 
-#endif
+class InputFile {
```

Comment:
Yep, I removed it in #23, both this file and `example_util.cc` are removed there